test/LaunchDarkly.EventSource.Tests/EventSourceEndToEndTest.cs

```diff
@@ -1,184 +0,0 @@
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Net;
-using System.Text;
-using System.Threading;
-using System.Threading.Tasks;
-using EmbedIO;
-using EmbedIO.Routing;
-using Xunit;
-using Xunit.Abstractions;
-
-namespace LaunchDarkly.EventSource.Tests
-{
-    // For most of the EventSource tests, it's adequate to use a fake HttpMessageHandler that provides
-    // the desired response data without really doing HTTP. But we do want to be sure that our I/O logic
-    // works as intended with a real HTTP client, so this class does some basic EventSource tests
-    // against an embedded HTTP server.
-
-    public class EventSourceEndToEndTest : BaseTest
```

Comment:
I moved these tests into `EventSourceHttpBehaviorTest` because they are _all_ end-to-end tests now.